tests/baselines/reference/visibilityOfCrossModuleTypeUsage.types

```diff
@@ -18,16 +18,16 @@ function run(configuration: commands.IConfiguration) {
 >                      : ^^^^^^
 >configuration.workspace.toAbsolutePath(configuration.server) : string
 >                                                             : ^^^^^^
->configuration.workspace.toAbsolutePath : (server: IServer, workspaceRelativePath?: string) => string
->                                       : ^      ^^       ^^                     ^^^      ^^^^^^^^^^^
+>configuration.workspace.toAbsolutePath : (server: import("visibilityOfCrossModuleTypeUsage_server").IServer, workspaceRelativePath?: string) => string

```

Comment:
Huh, interesting; I guess this was a bug before?

Reply:
`trackExistingEntityName` would previously check if the symbol could be resolved in the original context and if the symbol is accessible from the enclosing declaration. The answer to both of those is yes. The issue is that the symbol is not accessible in the *same way*. This can lead to the scope confusion I showed in one of the bugs. 

What I do now is I test if the symbol is also resolvable in the context it is being used in. And in this case `IServer` is not directly accessible so it falls back on `typeToTypeNodeHelper` which will print the import. 

I have a follow on PR that will try to find a more appropriate name in the enclosing declaration where this is printed (such as `commands.IServer` in this case) and will also try to use more of the original node even if sometimes this means replacing parts of the node with nodes printed from types.